.github/workflows/py-workflow.yml

```diff
@@ -159,6 +189,8 @@ jobs:
           -Dsonar.scm.provider=git
           -Dsonar.qualitygate.wait=true
           -Dsonar.qualitygate.timeout=300
+          -Dsonar.python.flake8.reportPaths=flake8-report.txt
+          -Dsonar.python.pylint.reportPaths=pylint-report.txt
```

Comment:
what happen if those reports are not present ?

Reply:
I haven't tested that since the step that generates the reports always runs so the reports will always exist